build.cmd

```diff
@@ -0,0 +1,16 @@
+<Project>
+    <ItemGroup>
+        <VersionFile Include="version"/>
+    </ItemGroup>
+    <PropertyGroup>
+        <NuspecPath>Microsoft.jQuery.Unobtrusive.Validation.nuspec</NuspecPath>
+    </PropertyGroup>
+    <Target Name="Build">
+        <ReadLinesFromFile File="@(VersionFile)">
+            <Output TaskParameter="Lines" PropertyName="PackageVersion"/>
+        </ReadLinesFromFile>
+        <Exec Command="npm version --no-git-tag-version --allow-same-version $(PackageVersion)" />
```

Comment:
@jbagga said:

> If the "version" file is updated it picks up that version or stays the same.

So it *won't* update any files on disk, even if the version has been updated? I just want to make sure that the build process doesn't modify files that are checked in, because then someone might check in the overwritten file, which would be wrong.